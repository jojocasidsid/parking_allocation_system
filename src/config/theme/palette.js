const blue = '#002F5D';
const black = '#282B2F';
const white = '#FFFFFF';

const exportColors = {
  colors: {
    blue,
    black,
    white
  },
  primary: {
    main: blue
  },
  secondary: {
    main: black
  },
  info: {
    main: white
  },
  text: {
    primary: black,
    secondary: '#B5B5B5'
  },
  action: {
    disabled: black
  }
};

export default exportColors;
