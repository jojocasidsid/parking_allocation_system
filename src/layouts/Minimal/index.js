import React from 'react';
import { Outlet } from 'react-router-dom';

const Minimal = () => (
  <div>
    <Outlet />
  </div>
);

export default Minimal;
