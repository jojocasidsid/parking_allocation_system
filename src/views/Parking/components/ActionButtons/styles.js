import styled from '@emotion/styled';

export const ActionWrapper = styled('div')`
  display: flex;
  justify-content: space-around;
  align-items: center;
  padding: 20px;
`;
