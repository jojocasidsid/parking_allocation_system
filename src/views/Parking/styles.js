import styled from '@emotion/styled';

export const SlotsWrapper = styled('div')`
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  grid-template-rows: repeat(5, 1fr);
  grid-gap: 10px;
`;

export const ActionWrapper = styled('div')`
  display: flex;
  justify-content: space-around;
  align-items: center;
  padding: 20px;
`;
