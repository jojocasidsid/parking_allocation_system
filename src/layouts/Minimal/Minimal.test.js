import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from '@mui/material/styles';

import theme from 'config/theme/theme';

import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';

import Minimal from '.';

describe('Minimal layout', () => {
  it('should render without crashing', () => {
    const tree = renderer.create(
      <ThemeProvider theme={theme}>
        <Router>
          <Routes>
            <Route path="*" element={<Minimal />} />
          </Routes>
        </Router>
      </ThemeProvider>
    );

    expect(tree).toMatchSnapshot();
  });
});
