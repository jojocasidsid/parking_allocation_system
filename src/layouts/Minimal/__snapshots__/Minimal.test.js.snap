// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Minimal layout should render without crashing 1`] = `<div />`;
